server/odc-core/src/main/resources/i18n/ErrorMessages.properties

```diff
@@ -186,6 +186,8 @@ com.oceanbase.odc.ErrorCodes.ExecPLByAnonymousBlockUndefinedParameters=Unable to
 com.oceanbase.odc.ErrorCodes.ExecPLByAnonymousBlockErrorFormatParameters=The parameter passing method in line {0} does not conform to the definition. The parameter must be passed using the p => p format. Please do not modify the generated template.
 com.oceanbase.odc.ErrorCodes.ExecPLByAnonymousBlockDifferentParameters=The PL parameter {1} in line {0} is inconsistent with the assignment content {2}. Please do not delete or modify the generated template.
 com.oceanbase.odc.ErrorCodes.ExecPLByAnonymousBlockErrorCallStatement=The call to the PL object {0} cannot be found in the anonymous block. Please do not modify the generated anonymous block template.
+com.oceanbase.odc.ErrorCodes.PartitionPlanNoCreatePreviewSqlGenerated=No partition creation statement is generated. Possible reasons: The newly generated partition is a duplicate of the existing partition.
+com.oceanbase.odc.ErrorCodes.PartitionPlanNoDropPreviewSqlGenerated=No partition deletion statement is generated. Possible reason: No partition matches the deletion rule.
```

Comment:
The SQL statement for partition deletion is not generated. Possible cause: No partition matches the deletion rule.

Reply:
solve later